src/app/retrospective/retrospective-create/retrospective-create.component.ts

```diff
@@ -96,6 +101,31 @@ export class RetrospectiveCreateComponent implements OnInit, OnDestroy {
             );
     }
 
+    getTimeProviders() {
+        const team: string = this.retroFormGroup.value.team;
+        let selectedTaskProvider: string;
+        this.retroFormGroup.value.taskProvider.forEach(provider => {
+            selectedTaskProvider = provider.selectedTaskProvider;
+        });
+
+        if (!team || !selectedTaskProvider) {
+            return;
+        }
+        this.disableTimeProviderField = false;
+        this.retrospectiveService.getTimeProvidersList(selectedTaskProvider, team)
+        .subscribe(
+            (data) => {
+                this.timeProvidersList = data.body.timeProviderList;
+                this.selectedTimeField = this.timeProvidersList[0];
```

Comment:
Shouldn't we set the team level time provider as the pre-selected value?

Reply:
discussed.